packages/generators/app/src/create-project.ts

```diff
@@ -12,6 +12,9 @@ import mergeTemplate from './utils/merge-template.js';
 import tryGitInit from './utils/git';
 
 import packageJSON from './resources/json/common/package.json';
+import jsconfig from './resources/json/js/jsconfig.json';
+import adminTsconfig from './resources/json/ts/tsconfig-admin.json';
+import serverTsconfig from './resources/json/ts/tsconfig-server.json';
```

Comment:
we import these now so they're bundled as using `fs` isn't ideal in a bundle world